crypto/rc4/asm/rc4-x86_64.pl

```diff
@@ -529,6 +537,7 @@ sub RC4_loop {
 .type	RC4_options,\@abi-omnipotent
 .align	16
 RC4_options:
+	$endbr
```

Comment:
hmm.  This function misses both .cfi_startproc and .cfi_endproc,
as well as the .size decoration, could you please add those here as well?

Reply:
I will submit a separate patch.